build.cmd

```diff
@@ -120,8 +120,14 @@ goto :eof
 :AfterBuild
 
 echo.
-:: Pull the build summary from the log file
-findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
-echo [%time%] Build Exit Code = %BUILDERRORLEVEL%
-
+if NOT [%ERRORLEVEL%]==[0] (
+  :: Pull the build summary from the log file
+  findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%">&2
+  echo [%time%] Build Exit Code = %BUILDERRORLEVEL%>&2
+  echo ERROR: An error occurred while building, see %_buildlog% for more details.>&2
+) else (
+  :: Pull the build summary from the log file
+  findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
```

Comment:
I wouldn't duplicate this. I would always just send that to the std out and only send the error message to std err.